gneiss/plot/_regression_plot.py

```diff
@@ -0,0 +1,369 @@
+                     header=True, index=True)
+    pvalues = model.pvalues
+    pvalues.to_csv(os.path.join(output_dir, 'pvalues.csv'),
+                   header=True, index=True)
+    balances = model.balances
+    balances.to_csv(os.path.join(output_dir, 'balances.csv'),
+                    header=True, index=True)
+
+
+def _deposit_results_html(index_f):
+    """ Create links to all of the regression results. """
+    index_f.write(('<th>Coefficients</th>\n'))
+    index_f.write(('<a href="coefficients.csv">'
+                   'Download as CSV</a><br>\n'))
+    index_f.write(('<th>Coefficient pvalues</th>\n'))
+    index_f.write(('<a href="pvalues.csv">'
+                   'Download as CSV</a><br>\n'))
+    index_f.write(('<th>Raw Balances</th>\n'))
+    index_f.write(('<a href="balances.csv.csv">'
+                   'Download as CSV</a><br>\n'))
+    index_f.write(('<th>Predicted Balances</th>\n'))
+    index_f.write(('<a href="predicted.csv">'
+                   'Download as CSV</a><br>\n'))
+    index_f.write(('<th>Residuals</th>\n'))
+    index_f.write(('<a href="residuals.csv">'
+                   'Download as CSV</a><br>\n'))
+    index_f.write(('<th>Tree</th>\n'))
+
+
+# OLS summary
+def ols_summary(output_dir: str, model: OLSModel,
+                tree: TreeNode) -> None:
+    """ Summarizes the ordinary least squares fit.
+
+    Parameters
+    ----------
+    output_dir : str
+        Directory where all of the regression results and
+        summaries will be stored.
+    model : OLSModel
+        Ordinary Least Squares model that contains the model fit and the
+        regression results.
+    tree : TreeNode
+        Tree object that defines the partitions of the features. Each of the
+        leaves correspond to the balances in the model.
+    """
+    # Cross validation
+    cv = model.loo()
+    w, h = 300, 300  # plot width and height
+
+    # Explained sum of squares
+    ess = pd.Series({r.model.endog_names: r.ess for r in model.results})
+
+    # Summary object
+    _k, _l = model.kfold(), model.lovo()
+    smry = model.summary(_k, _l)
+    _deposit_results(model, output_dir)
+    t = _decorate_tree(tree, ess)
+
+    p1 = radialplot(t, edge_color='color', figsize=(800, 800))
+    p1.title.text = 'Explained Sum of Squares'
+    p1.title_location = 'above'
+    p1.title.align = 'center'
+    p1.title.text_font_size = '18pt'
+
+    # 2D scatter plot for prediction on PB
+    p2 = _projected_prediction(model, plot_width=w, plot_height=h)
+    p3 = _projected_residuals(model, plot_width=w, plot_height=h)
+    hm_p = _heatmap_summary(model.pvalues, model.coefficients())
+
+    # combine the cross validation, explained sum of squares tree and
+    # residual plots into a single plot
+    p = row(column(p2, p3), p1)
+    p = column(hm_p, p)
+    index_fp = os.path.join(output_dir, 'index.html')
+    with open(index_fp, 'w') as index_f:
+        index_f.write('<html><body>\n')
```

Comment:
So the HTML page does not have `<head>`. Although not required, it is favorable to put things like `<title>` and external resources in it.